docs/next/styles/globals.css

```
@@ -104,6 +104,11 @@ dt > a.reference.internal {
   @apply inline-flex items-center px-3 py-0.5 rounded-full align-middle text-xs uppercase font-medium bg-sea-foam text-gable-green
 }
 
+.preview-tag,
+.flag.preview{
```

Linter should probably catch this, but

```suggestion
.flag.preview {
```

---

Good catch, thanks!